index.js

```diff
@@ -4,9 +4,24 @@ let instances = {};
 
 function DB() {
   this.pool = null;
+  this.sessionConfig = null;
   this.debug = false;
 }
 
+function setSessionConfig(connection, config) {
+  if(!config) {
+    return;
+  }
+
+  if(this.debug) {
+    console.log(`Set Session Config ${config}`);
```

Comment:
Actually, in JS, this will print `Set Session Config [object Object]`. It should be `console.log(`Set Session Config ${JSON.stringify(config)}`);`.

Reply:
Not really. console.log implementation in nodejs uses another utility (util.inspect) to format the given parameters by default, it shows the json representation text of the given parameters (by default it shows nested objects to the second depth, after that it would show [Object Object]).